src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml

```diff
@@ -0,0 +1,144 @@
+                            <th scope="col" class="border-left pl-3">Approved last</th>
+                            <th scope="col" class="border-left pl-3">Approved by</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @if (Model.ApprovedReviews.Any())
+                        {
+                            @foreach (var review in Model.ApprovedReviews)
+                            {
+                                var truncationIndex = @Math.Min(@review.DisplayName.Length, 100);
+                                <tr class="package-data-row">
+                                    <td class="align-middle pl-4 pt-0 pb-0">
+                                        @if (review.Language != null)
+                                        {
+                                            string iconClassName = "icon-" + review.GetLanguageCssSafeName();
+                                            @if (!string.IsNullOrEmpty(review.LanguageVariant) && review.LanguageVariant.ToLower() != "default")
+                                            {
+                                                iconClassName += "-" + @review.LanguageVariant.ToLower();
+                                            }
+                                            <span role="img" class="mx-1 icon icon-language @iconClassName" aria-label="@review.Language @review.LanguageVariant"></span>
+                                        }
+                                        <a class="review-name align-middle" asp-page="./Review" asp-route-id="@review.ReviewId">@review.DisplayName.Substring(0, @truncationIndex)</a>
+                                        @if (review.IsApproved == true)
+                                        {
+                                            <i class="fas fa-check-circle text-success ml-2"></i>
+                                        }
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <a username="@review.Author">@review.Author</a>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span data-placement="left" data-toggle="tooltip" title="@review.LastUpdated" date="@review.LastUpdated.ToLocalTime()"></span>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span>@review.FilterType.ToString()</span>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span data-placement="left" data-toggle="tooltip" title="@review.LastUpdated" date="@review.ApprovalDate.ToLocalTime()"></span>
```

Comment:
This will fail if approval date is null which is the case of existing approved reviews. right? We need to handle the scenario when review doesn't have approval date (new field)

Reply:
This will only happen if a review is requested for an already approved API, but the latest commit also will filter out the cases where the review is requested and no date is listed for approval